roxctl/common/k8s_port_forward.go

```diff
@@ -108,22 +110,34 @@ func getCentralAPIPort(pod *corev1.Pod) int32 {
 	return defaultAPIPort
 }
 
-func getPortForwarder(restConfig *rest.Config, pod *corev1.Pod, stopChannel <-chan struct{}, readyChannel chan struct{}) (*portforward.PortForwarder, error) {
+func getCentralCA(ctx context.Context, coreClient coreclient.CoreV1Interface, namespace string) ([]byte, error) {
+	centralTLS, err := coreClient.Secrets(namespace).Get(ctx, "central-tls", metav1.GetOptions{})
+	if err != nil {
+		return nil, err //nolint:wrapcheck
+	}
+	if ca, ok := centralTLS.Data["ca.pem"]; ok {
+		return ca, nil
+	}
+	return nil, nil
+}
+
+func getPortForwarder(restConfig *rest.Config, pod *corev1.Pod, stopChannel <-chan struct{}, readyChannel chan struct{}) (uint16, *portforward.PortForwarder, error) {
 	restClient, err := rest.RESTClientFor(restConfig)
 	if err != nil {
-		return nil, errors.WithMessage(err, "failed to construct k8s REST client")
+		return 0, nil, errors.WithMessage(err, "failed to construct k8s REST client")
 	}
 	req := restClient.Post().Resource(corev1.ResourcePods.String()).
 		Namespace(pod.GetNamespace()).Name(pod.GetName()).
 		SubResource("portforward")
```

Comment:
Think it would be great to enumerate the required RBAC for using the flag (e.g. access to port forward in the stackrox namespace).